src/Measures.jl

```diff
@@ -558,3 +559,55 @@ function get_redundancy(xyz; estimator = "maximum_likelihood", base = 2, probabi
 		return apply_redundancy_formula(probabilities_xz, probabilities_yz, probabilities_x, probabilities_y, probabilities_z, (1, 2), base)
 	end
 end
+
+
+"""
+    cross_entropy(values_x, values_y; <keyword arguments>)
+Estimate the cross-entropy between two sets of values.
+# Arguments:
+* `values_x`: the first set of data values.
+* `values_y`: the second set of data values.
+* `estimator="maximum_likelihood"`: the entropy estimator.
+* `base=2`: the base of the logarithm, equivalent to the units of information.
+* `mode="uniform_width"`: the discretization algorithm.
+* `number_of_bins=0`: the number of bins (will be overridden if `mode` is `"bayesian_blocks"`).
+* `get_number_of_bins=get_root_n`: the method for calculating the number of bins (only called if `number_of_bins` is `0`).
+* `discretized=false`: whether the data values are already discretized.
+* `lambda=nothing`: the shrinkage instensity, only used if `estimator` is `"shrinkage"`.
+* `prior=1`: the Dirichlet prior, only used if `estimator` is `"dirichlet"`.
+"""
+function cross_entropy(
+        values_x,
+        values_y;
+        estimator="maximum_likelihood",
+        base=2,
+        mode="uniform_width",
+        number_of_bins=0,
+        get_number_of_bins=get_root_n,
+        discretized=false,
+        lambda=nothing,
+        prior=1,
+        )
+    frequency_x, frequency_y = discretized ?
```

Comment:
This works great in Julia 0.6, but the dot calls work differently in Julia 0.5. (We're still supporting Julia 0.5, though maybe not for much longer...) Could you do it like above instead for now?